package.json

```diff
@@ -301,6 +301,9 @@
     "sinon": "7.5.0",
     "style-loader": "0.23.1",
     "styled-components": "5.3.0",
+    "trezor-connect": "8.1.5-extended",
+    "sharedworker-loader": "^2.1.1",
+    "worker-loader": "2.0.0",
```

Comment:
Please add a comment as to why we needed to add these even though they aren't directly referenced by our code. Also remove the `^` prefix, as we seem to specify exact versions for all dependencies.

Reply:
done

Reply:
I don't see the comment, how about adding the reason to the git commit message for this PR? That would be helpful for me. Otherwise I'd see the dependency, see that it's not directly used and remove it.

Reply:
done